packages/performance/wafer-map/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { waferMapTag } from '@ni/nimble-components/dist/esm/wafer-map';
+import type { WaferMapDie } from '@ni/nimble-components/dist/esm/wafer-map/types';
+
+document.addEventListener('DOMContentLoaded', main);
+
+async function waitAnimationFrame(): Promise<void> {
+    return new Promise(resolve => {
+        requestAnimationFrame(() => resolve());
+    });
+}
+
+async function runBenchmark(benchmark: string, dieCount: number): Promise<void> {
+    const waferMapDies = generateDies(dieCount);
+
+    const start = performance.now();
+    const waferMap = document.createElement(waferMapTag);
+    waferMap.dies = waferMapDies;
+    waferMap.colorScale = {
+        colors: ['red', 'orange', 'yellow', 'green'],
+        values: ['1', '33', '66', '100']
+    };
+    document.body.append(waferMap);
+    await waitAnimationFrame();
+    performance.measure(benchmark, { start });
+
+    document.body.removeChild(waferMap);
```

Comment:
I made sure to clean-up the previous wafer-map before running the next test to avoid the chance that the wafer-maps will interfere with each other during benchmarking